### ISSUE TITLE
[Accessibility] Narrator announces state as ‘Expanded’ for the collapsed left navigation pane items.

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#1862]
**Test Environment:**
OS Build: Version 22H2(OS Build 25140.1000) 
Browser: Version 104.0.1278.2 (Official build) dev (64-bit)
URL: [Graph Explorer | Try Microsoft Graph APIs - Microsoft Graph](https://developer.microsoft.com/en-us/graph/graph-explorer)
Screen reader: Narrator

**To Reproduce**
1. Turn on Narrator using ‘Win + Ctrl + Enter’ key.
2. Open the above [URL](https://developer.microsoft.com/en-us/graph/graph-explorer) in Edge browser.
3. Navigate the left navigation pane via down arrow key,
4. Observe the issue.

**Actual Result:**
When the user navigates the left navigation pane, narrator announces the state as expanded, even if visually the item is in collapsed state.

**Expected Result:**
The narrator should announce correct state for the left navigation pane items.
If visually the item in collapsed state, the narrator should announce it as collapsed. And same for Expanded state.

**Observation:**
Same issue is not repro with NVDA

**User Impact:**
The PWD users will be confused if narrator announces incorrect information to the user.

**MAS Reference:**
[Info and Relationships - Liquid (microsoft.com)](https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.01)

**Additional context**
Issue reference: https://office.visualstudio.com/OC/_workitems/edit/6175859/
